products/accesscontextmanager/api.yaml

```diff
@@ -260,3 +260,154 @@ objects:
+      GCP project can only belong to a single regular Service Perimeter. Service
+      Perimeter Bridges can contain only GCP projects as members, a single GCP
+      project may belong to multiple Service Perimeter Bridges.
+<%= indent(compile_file({}, 'templates/global_async.yaml.erb'), 4) %>
+    parameters:
+      # Parent is a path parameter that _cannot_ be read or sent in the request at all.
+      # This must be done at the provider level.
+      - !ruby/object:Api::Type::String
+        name: parent
+        input: true
+        required: true
+        description: |
+          The AccessPolicy this ServicePerimeter lives in.
+          Format: accessPolicies/{policy_id}
+      - !ruby/object:Api::Type::String
+        name: name
+        input: true
+        required: true
+        description: |
+          Resource name for the ServicePerimeter. The short_name component must
+          begin with a letter and only include alphanumeric and '_'.
+          Format: accessPolicies/{policy_id}/servicePerimeters/{short_name}
+    properties:
+      - !ruby/object:Api::Type::String
+        name: title
+        required: true
+        description: |
+          Human readable title. Must be unique within the Policy.
+      - !ruby/object:Api::Type::String
+        name: 'description'
+        description: |
+          Description of the ServicePerimeter and its use. Does not affect
+          behavior.
+      - !ruby/object:Api::Type::Time
+        name: 'createTime'
+        description: |
+          Time the AccessPolicy was created in UTC.
+        output: true
+      - !ruby/object:Api::Type::Time
+        name: 'updateTime'
+        description: |
+          Time the AccessPolicy was updated in UTC.
+        output: true
+      - !ruby/object:Api::Type::Enum
+        name: 'perimeterType'
+        description: |
+          Specifies the type of the Perimeter. There are two types: regular and
+          bridge. Regular Service Perimeter contains resources, access levels,
+          and restricted/unrestricted services. Every resource can be in at most
+          ONE regular Service Perimeter.
+
+          In addition to being in a regular service perimeter, a resource can also
+          be in zero or more perimeter bridges. A perimeter bridge only contains
+          resources. Cross project operations are permitted if all effected
+          resources share some perimeter (whether bridge or regular). Perimeter
+          Bridge does not contain access levels or services: those are governed
+          entirely by the regular perimeter that resource is in.
+
+          Perimeter Bridges are typically useful when building more complex
+          toplogies with many independent perimeters that need to share some data
+          with a common perimeter, but should not be able to share data among
+          themselves.
+        values:
+          - :PERIMETER_TYPE_REGULAR
+          - :PERIMETER_TYPE_BRIDGE
```

Comment:
Can you add a test for `PERIMETER_TYPE_BRIDGE` ?

Reply:
Done